packages/components/navigation/Sidebar.tsx

```diff
@@ -83,6 +83,7 @@ export const Sidebar: React.FC = () => {
   );
 
   const onRouteChange = (name: SidebarType["route"]) => {
+    // @ts-ignore
```

Comment:
please always use ts-expect-error instead of ts-ignore
and in general please avoid ignoring ts errors, we want type safety